pages/utils/sum.ts

```diff
@@ -0,0 +1,8 @@
+export function sum(a: number, b: number, format: "html" | "text" = "html") {
```

Comment:
Maybe it's best to rename the a and b parameter to "num1" or "num2" for example to make much more sense compared to just a and b?

Reply:
Duplicate of https://github.com/reactcebu/code-review-practice/pull/2#discussion_r1110816731. Resolving...